R/cbio_setup_study.R

```diff
@@ -123,7 +123,7 @@ cbio_setup_study = function(
     } else {
         coding_ssms <- maf_data %>%
             dplyr::filter(
-                Variant_Classification %in% GAMBLR.helpers:::coding_class
+                Variant_Classification %in% GAMBLR.helpers::coding_class
```

Comment:
This small change is because `:::` is used when a object is no exported from the package. But `coding_class` is exported from `GAMBLR.helpers`.

Reply:
I thought this was exported. We should make that available as global variable in helpers so it can be used here

Reply:
`coding_class` is an exported object from GAMBLR.helpers, so we just need a `::` to use it here in GAMBLR.results. If it wasn't marked with the `@export` tag, we would need `:::`.

Reply:
right so this change is not needed since the coding_class is an exported variable